presto/dataset.py

```diff
@@ -308,6 +319,10 @@ def __getitem__(self, idx):
         df_index = self.indices[idx]
         row = self.df.iloc[df_index, :]
         eo, mask_per_token, latlon, month, target = self.row_to_arrays(row, self.target_function)
+        if self.mask_ratio > 0:
+            mask_per_token, eo, _, _ = self.mask_params.mask_data(
+                self.normalize_and_mask(eo), mask_per_token
```

Comment:
Don't call normalize and mask here, it gets called later

Reply:
Regarding this comment, did you double check that? `normalize_and_mask` still called on `eo` and when returning from the function `normalize_and_mask` called once again? This doesn't do double normalization?

Reply:
thanks for catching this! i left this comment intending to remove this call and forgot 🙈

Reply:
fixed